iquail/helper/__init__.py

```diff
@@ -3,3 +3,4 @@
 from .integrity_verifier import IntegrityVerifier, checksum_file
 from .configuration import Configuration, ConfVar
 from .linux_polkit_file import polkit_install, polkit_check
+from .osx import BundleTemplate, PlistCreator
```

Comment:
J'ai deja dit, mais juste pour être certain: ça va pas importer de la merde juste compatible mac hein?

Reply:
pListCreator c'est un [utilitaire XML](https://docs.python.org/3/library/xml.etree.elementtree.html). BundleTemplate est juste un util de creation de dossier.

Reply:
ca serait quand meme cool d'importer l'helper osx seulement sur osx

Reply:
Carrement

Reply:
Fixed in last commit